### PR TITLE
Feature/464057 total bill 2c

### PR DESCRIPTION
**Pull request by udaydefra**

### CODE REVIEW

File: src/EPR.Calculator.API/Builder/Summary/TwoCCommsCost/TwoCCommsCostColumnIndex.cs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿namespace EPR.Calculator.API.Builder.Summary.TwoCCommsCost;
+
+public class TwoCCommsCostColumnIndex
+{
+    public const int Value = 203;
```
Comment (Mazar-Shaik) [inline src/EPR.Calculator.API/Builder/Summary/TwoCCommsCost/TwoCCommsCostColumnIndex.cs:5]: For one property do we need separate class? Can we add this in the same class or all sections indexes into one class which looks good. If other devs also follow this single class pattern then we end up with multiple classes. 
What do you say?

**Review by johnbenjamin-defra — APPROVED** [review]

File: src/EPR.Calculator.API/Builder/Summary/CalcResultSummaryBuilder.cs [hunk 2]
```diff
@@ -110,107 +125,137 @@ private static CalcResultSummaryProducerDisposalFees GetProducerTotalRow(
...
+                LaDataPrepCostsScotlandTotalWithBadDebtProvisionSection4 = CalcResultSummaryUtil.GetLaDataPrepCostsScotlandTotalWithBadDebtProvisionSection4(),
+                LaDataPrepCostsNorthernIrelandTotalWithBadDebtProvisionSection4 = CalcResultSummaryUtil.GetLaDataPrepCostsNorthernIrelandTotalWithBadDebtProvisionSection4(),
+
+                // Percentage of Producer Reported Household Tonnage vs All Producers
+                PercentageofProducerReportedHHTonnagevsAllProducers = HHTonnageVsAllProducerUtil.GetPercentageofProducerReportedHHTonnagevsAllProducersTotal(producersAndSubsidiaries, runProducerMaterialDetails),
+                
                 isTotalRow = true
             };
 
+            TwoCCommsCostUtil.UpdateTwoCTotals(calcResult, producerDisposalFees, isOverAllTotalRow, totalRow);
```
Comment (Mazar-Shaik) [inline src/EPR.Calculator.API/Builder/Summary/CalcResultSummaryBuilder.cs:240]: Can this be implemented like line number 220 to 223 to make consistent all over?

File: src/EPR.Calculator.API/Builder/Summary/CalcResultSummaryBuilder.cs [hunk 3]
```diff
@@ -219,106 +264,129 @@ private static CalcResultSummaryProducerDisposalFees GetProducerRow(
...
+                LaDataPrepCostsEnglandTotalWithBadDebtProvisionSection4 = CalcResultSummaryUtil.GetLaDataPrepCostsEnglandTotalWithBadDebtProvisionSection4(),
+                LaDataPrepCostsWalesTotalWithBadDebtProvisionSection4 = CalcResultSummaryUtil.GetLaDataPrepCostsWalesTotalWithBadDebtProvisionSection4(),
+                LaDataPrepCostsScotlandTotalWithBadDebtProvisionSection4 = CalcResultSummaryUtil.GetLaDataPrepCostsScotlandTotalWithBadDebtProvisionSection4(),
+                LaDataPrepCostsNorthernIrelandTotalWithBadDebtProvisionSection4 = CalcResultSummaryUtil.GetLaDataPrepCostsNorthernIrelandTotalWithBadDebtProvisionSection4(),
+
+                // Percentage of Producer Reported Household Tonnage vs All Producers
+                PercentageofProducerReportedHHTonnagevsAllProducers = HHTonnageVsAllProducerUtil.GetPercentageofProducerReportedHHTonnagevsAllProducers(producer, runProducerMaterialDetails),
             };
+
+            TwoCCommsCostUtil.UpdateTwoCRows(calcResult, result);
```
Comment (Mazar-Shaik) [inline src/EPR.Calculator.API/Builder/Summary/CalcResultSummaryBuilder.cs:374]: Can this be implemented like line number 356 to 359 to make consistent all over?

Comment (udaydefra) [reply src/EPR.Calculator.API/Builder/Summary/CalcResultSummaryBuilder.cs]: Done

**Review by Mazar-Shaik — DISMISSED** [review]

**Review by johnbenjamin-defra — DISMISSED** [review]

**Comment by udaydefra** [comment]
https://github.com/DEFRA/epr-calculator-api/pull/121

**Pull request closed without merging**